tests/scripts/thread-cert/border_router/test_manual_omr_prefix.py

```diff
@@ -79,21 +79,29 @@ def test(self):
 
         # Add a smaller OMR prefix. Verify BR_1 withdraws its OMR prefix.
         self._test_manual_omr_prefix('2001::/64', 'paros', expect_withdraw=True)
+        # Add a bigger OMR prefix. Verify BR_1 does not withdraw its OMR prefix.
+        self._test_manual_omr_prefix('fdff:ffff:1::/64', 'paros', expect_withdraw=False)
```

Comment:
nit: better to make the difference explicit because this is what exactly causing different results.
```suggestion
        self._test_manual_omr_prefix('fdff:ffff:1::/64', 'paros', expect_withdraw=False, prf='med')
```